### DISCUSSION TITLE
CoolProp for Heat Transfer Fluid—Therminol VP-1

### DISCUSSION THREAD

**Discussion by almarry9** [CoolProp/CoolProp#2287] [General]
I'm just wondering if CoolProp has the Heat Transfer Fluid (Therminol VP-1). I tried this code [py.CoolProp.CoolProp.PropsSI('H','T',120+273.15,'P',1280,'TVP1')].
However, it did not work.

Thank you and I appreciate your collaboration.

**Comment by ibell** [comment]
You need to specify the INCOMP backend, so it would be:

```matlab
py.CoolProp.CoolProp.PropsSI('H','T',120+273.15,'P',1280,'INCOMP::TVP1')
```

**Reply by almarry9** [reply]
Thank you. Now, I know what is the problem.
when I used this:
py.CoolProp.CoolProp.PropsSI('H','T',120+273.15,'P',1280,'INCOMP::TVP1')

It gives me this error:
Python Error: ValueError: Equations are valid for liquid phase only: 1280.000000 < 1281.332896 (psat)

So I used this and it worked:
py.CoolProp.CoolProp.PropsSI('H','T',120+273.15,'P',2000,'INCOMP::TVP1')
ans = 
169020 J/kg

Thank you again
